package.json

```diff
@@ -34,69 +34,68 @@
     "build:watch:esm": "tsc -p tsconfig.esm.json -w --preserveWatchOutput",
-    "clean": "rimraf coverage esm main preload renderer index.* integrations.* ipc.*",
-    "preexample": "run-s clean build",
-    "example": "electron example",
+    "clean": "rimraf coverage esm main preload renderer index.* integrations.* ipc.* sentry-electron*.tgz",
     "lint": "run-s lint:prettier lint:eslint",
     "lint:prettier": "prettier --check \"{src,test}/**/*.ts\"",
     "lint:eslint": "eslint . --cache --format stylish",
     "fix": "run-s fix:eslint fix:prettier",
     "fix:prettier": "prettier --write \"{src,test}/**/*.ts\"",
     "fix:eslint": "eslint . --format stylish --fix",
-    "pretest": "run-s clean build",
+    "pretest": "yarn build",
     "test": "cross-env TS_NODE_PROJECT=tsconfig.json xvfb-maybe electron-mocha --require ts-node/register/transpile-only --timeout 120000 ./test/unit/**/*.ts",
-    "pree2e": "run-s clean build",
-    "e2e": "cross-env TS_NODE_PROJECT=tsconfig.json xvfb-maybe mocha --require ts-node/register/transpile-only --timeout 30000 --retries 3 ./test/e2e/*.ts"
+    "pree2e": "rimraf test/e2e/dist/**/node_modules/@sentry/electron/** test/e2e/dist/**/yarn.lock && node scripts/clean-cache.js && yarn build && npm pack",
+    "e2e": "cross-env TS_NODE_PROJECT=tsconfig.json xvfb-maybe mocha --require ts-node/register/transpile-only --retries 3 ./test/e2e/*.ts"
   },
   "dependencies": {
-    "@sentry/browser": "6.13.2",
-    "@sentry/core": "6.13.2",
-    "@sentry/node": "6.13.2",
-    "@sentry/types": "6.13.2",
-    "@sentry/utils": "6.13.2",
+    "@sentry/browser": "6.13.3",
+    "@sentry/core": "6.13.3",
+    "@sentry/node": "6.13.3",
+    "@sentry/types": "6.13.3",
+    "@sentry/utils": "6.13.3",
     "deepmerge": "^4.2.2",
-    "tslib": "^2.3.0"
+    "tslib": "^2.3.1"
   },
   "devDependencies": {
-    "@sentry-internal/eslint-config-sdk": "6.13.2",
-    "@sentry-internal/typescript": "6.13.2",
-    "@sentry/tracing": "6.13.2",
+    "@sentry-internal/eslint-config-sdk": "6.13.3",
+    "@sentry-internal/typescript": "6.13.3",
+    "@sentry/tracing": "6.13.3",
     "@types/busboy": "^0.2.3",
     "@types/chai": "^4.2.10",
     "@types/chai-as-promised": "^7.1.0",
-    "@types/chai-datetime": "^0.0.33",
+    "@types/chai-subset": "^1.3.3",
     "@types/form-data": "^2.5.0",
     "@types/koa": "^2.0.52",
     "@types/koa-bodyparser": "^4.3.0",
-    "@types/mocha": "^5.2.5",
+    "@types/mocha": "^9.0.0",
     "busboy": "^0.3.1",
     "chai": "^4.1.2",
     "chai-as-promised": "^7.1.1",
-    "cross-env": "^5.2.0",
+    "chai-subset": "^1.6.0",
+    "cross-env": "^7.0.3",
     "eslint": "7.27.0",
-    "electron": "15.0.0",
-    "electron-download": "^4.1.1",
-    "electron-mocha": "^6.0.4",
-    "extract-zip": "^1.6.7",
-    "koa": "^2.13.1",
+    "electron": "15.3.0",
+    "@electron/get": "^1.13.0",
+    "electron-mocha": "^10.1.0",
+    "extract-zip": "^2.0.1",
+    "koa": "^2.13.4",
```

Comment:
Can we possibly move these deps into a e2e test package.json? Or too much trouble you think?

Reply:
Yes, it might be nice to split it into a workspace with SDK + Tests.

Shall I do that in another PR?

Reply:
Yup separate PR sounds like the best approach